src/main/java/com/minecraftheads/leathercolorizer/LeatherColorizer.java

```diff
@@ -56,6 +55,9 @@ private void checkConfig() {
             for (String language : LANGUAGES) {
                 String languageFile = "languages/" + language + ".yml";
 
+                File file = new File(getDataFolder(), languageFile);
+                if (file.exists()) continue;
```

Comment:
does this work?
i guess the configs would not get updated.
mabye do it like this?:
```
if (file.exists()) {
    ConfigUpdater.update(this, languageFile, configFile, Collections.emptyList());
} else {
    this.saveResource(languageFile, false);
}

Reply:
Yes indeed. Made a bug report for this already, this code fixes this